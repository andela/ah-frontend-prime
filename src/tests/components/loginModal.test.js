import { shallow } from "enzyme";
import React from "react";

import LoginComponent from "../../components/login/LoginModal";

describe("login modal success tests", () => {
  const props = {
    onSubmit: jest.fn(),
    emailError: "",
    passwordError: "",
    errors: null,
    generalError: "",
    onRegister: jest.fn(),
    onPassword: jest.fn()
  };
  it("should render without crashing", () => {
    const wraps = shallow(<LoginComponent {...props} />);
    expect(wraps).toMatchSnapshot();
  });
});

describe("login modal fail tests", () => {
  const props = {
    onSubmit: jest.fn(),
    emailError: "error",
    passwordError: "error",
    errors: {},
    generalError: "error"
  };
  it("should render without crashing", () => {
    const wraps = shallow(<LoginComponent {...props} />);
    expect(wraps).toMatchSnapshot();
  });
});
