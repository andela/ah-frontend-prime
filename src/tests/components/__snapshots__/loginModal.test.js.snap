// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login modal fail tests should render without crashing 1`] = `ShallowWrapper {}`;

exports[`login modal success tests should render without crashing 1`] = `ShallowWrapper {}`;
